requirements/dev.txt

```diff
@@ -19,6 +21,6 @@ sphinx==1.7.2
 sphinx-autobuild==0.7.1
 sphinx-markdown-tables==0.0.9
 sphinx-rtd-theme==0.3.0
+prometheus-client==0.7.1
```

Comment:
@nikochiko do we need this ?

Reply:
@yashdusing Yes, it is a dependency of dramatiq_sqs

Reply:
Reference:  https://github.com/Bogdanp/dramatiq/blob/master/setup.py#L42-L44

Reply:
@nikochiko could you also check the coverage ?